css/form.css

```diff
@@ -1054,10 +1054,6 @@ html[dir="rtl"] .radio.radio-icon input[type="radio"] + label > span.check {
   color: #ffffff;
 }
 
-[data-widget-package="com.fliplet.form-builder"] .form-group.fl-range-slider.readonly .range-slider-handle-value::before {
```

Comment:
Why is this removed if we're only adding validation to the configurations?

Reply:
removed un-necessary css which was already present in ui.rangeslider.css.

Reply:
@yogeshmsharma-xoriant Based on [this PR rejection](https://github.com/Fliplet/fliplet-api/pull/5599#pullrequestreview-1220772074), does this need to be added back?